app/controllers/organizations_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
   end
 
   def show
+  @movements = @organization.account.movements.order("created_at DESC").page(params[:page]).per(10)
```

Comment:
Account Load (136.6ms)  SELECT  "accounts".* FROM "accounts" WHERE "accounts"."accountable_id" = $1 AND "accounts"."accountable_type" = $2 LIMIT 1  [["accountable_id", 2], ["accountable_type", "Organization"]]
 
Account Load (136.6ms)  SELECT  "accounts".* FROM "accounts" WHERE "accounts"."accountable_id" = $1 AND "accounts"."accountable_type" = $2 LIMIT 1  [["accountable_id", 2], ["accountable_type", "Organization"]]